plugins/modem-manager/fu-mm-device.c

```diff
@@ -196,6 +196,41 @@ validate_firmware_update_method(MMModemFirmwareUpdateMethod methods, GError **er
 	return FALSE;
 }
 
+static void
+fu_mm_device_probe_quectel_flags(FuMmDevice *self)
+{
+	const gchar *version = fu_device_get_version(FU_DEVICE(self));
+	g_autofree gchar *name = NULL;
+	struct {
+		const gchar *name;
+		const gchar *version;
+	} secboot[] = {{"EM05GF", "EM05GFAR07A07M1G_01.005.01.005"},
```

Comment:
I'm sure you're not happy with having a hardcoded list; I guess there's no better way to handle this, right?

Reply:
Ya I'm not 100% happy, but apparently there's no way of asking the firmware "_hey are you signed_" -- at least not yet. Even some heuristic on the version string would be good, but alas.

Reply:
Are those earlier versions preproduction presumably? Maybe it's TOO much detail?
Especially if downgrades aren't supported.

Reply:
This has me questioning the efficacy of such an effort. If the boot rom isn't signed and the first payload you had in there wasn't signed how can you guarantee the safety of a payload that introduced signing?

Maybe that's too deep a question in root of trust and we should just be happy it's signed now and move on.

Reply:
The new firmware blows an eFuse!